Dockerfile-builder.template

```diff
@@ -265,8 +265,10 @@ RUN set -eux; \
 		CONFIG_FEATURE_UTMP \
 		CONFIG_FEATURE_WTMP \
 {{ ) else "" end -}}
+{{ if .version | startswith("1.35.") then ( -}}
 # disable SHA hardware acceleration (temporarily), as it fails with SIGILL on some of GitHub's common CI systems; https://bugs.busybox.net/show_bug.cgi?id=15236
 		CONFIG_SHA256_HWACCEL CONFIG_SHA1_HWACCEL \
```

Comment:
Fixed in https://git.busybox.net/busybox/commit/?h=1_36_stable&id=8d198665a0d09c3f22f3a12234809f46b1c2b04a :eyes: